### ISSUE TITLE
better regions repr

### ISSUE THREAD

**Issue by mathause** [mathause/regionmask#108]
the current is not very useful

**Comment by mathause** [comment]
For `Regions` it looks like

```python
import regionmask
regionmask.defined_regions.natural_earth.countries_110                                                                                                                                                 
```

```
177 'Natural Earth Countries: 110m' Regions (http://www.naturalearthdata.com)
FJ TZ WS CA US KZ UZ PG INDO AR CL DRC SO KE SD TD HT DO RUS BS FK N GL TF TL
ZA LS MX UY BR BO PE CO PA CR NI HN SV GT BZ VE GY SR F EC PR J CU ZW BW NA SN 
```

For `_one_Region`

```python
regionmask.defined_regions.natural_earth.countries_50["NL"]                  
```                                                                                                                           

```
Region: Netherlands (NL / 96)
center: [ 5.64213378 52.28215279]
```

Maybe something like this could be useful:
```
177 'Natural Earth Countries: 110m' Regions (http://www.naturalearthdata.com)
   0 Fiji     (FJ)
   1 Tanzania (TZ)
 ...
 176 S. Sudan (SS)
```

```
177 'Natural Earth Countries: 110m' Regions (http://www.naturalearthdata.com)
   0. Fiji     (FJ)
   1. Tanzania (TZ)
 ...
 176. S. Sudan (SS)
```

```
177 'Natural Earth Countries: 110m' Regions (http://www.naturalearthdata.com)
 - Fiji     (FJ / 0)
 - Tanzania (TZ / 1)
 - ...
 - S. Sudan (SS / 176)
```

```
177 'Natural Earth Countries: 110m' Regions (http://www.naturalearthdata.com)
 -   0 Fiji     (FJ)
 -   1 Tanzania (TZ)
 - ...
 - 176 S. Sudan (SS)
```

**Comment by mathause** [comment]
```
<regionmask.Regions>
Name:     Natural Earth Countries: 110m
Source:   http://www.naturalearthdata.com
Total:    177 regions

Regions:
  0. FJ (Fiji)
  1. TZ (Tanzania)
...
176. SS (S. Sudan)
```

```
<regionmask.Regions>
Name:     Natural Earth Countries: 110m
Source:   http://www.naturalearthdata.com

Individual regions:
       abbrev  (name)
  0.   FJI     (Fiji)
  1.   TZ      (A country with a very long name tha...)
..     ...     ...
176.   SS      (S. Sudan)

[177 regions]
```

**Comment by mathause** [comment]
Closed by #128
